src/applications/edu-benefits/0994/pages/highTechIndustry.js

```diff
@@ -52,27 +53,7 @@ export const uiSchema = {
       },
     },
     highTechnologyEmploymentType: {
-      'ui:title': ' ',
```

Comment:
Even though you're writing a custom field, it's been useful for us to still keep content like titles in the form config, since that's typically where someone familiar with our forms looks first for these fields. There's a `uiSchema` prop passed to any custom field that contains all this information in this object.

It looks like you could make that work with your custom field without too much trouble.